ui/app/routes/vault/cluster/secrets/backend/configuration/edit.ts

```diff
@@ -2,7 +2,8 @@
  * Copyright (c) HashiCorp, Inc.
  * SPDX-License-Identifier: BUSL-1.1
  */
-
+// eslint-disable-next-line
+// @ts-ignore
```

Comment:
Should we make tickets for the files with `@ts-ignore`? I could go either way - they'll be handled eventually (maybe) but maybe a ticket would help prioritize a little quicker - or at least keep track somewhere that some of our `ts` files are kind of just "typescript"

Reply:
Good suggestion but I gave `AdapterError` the same treatment as `Store` based on your comment and the types are being pulled in from the `@ember-data` namespace now so I will get rid of the ignore!